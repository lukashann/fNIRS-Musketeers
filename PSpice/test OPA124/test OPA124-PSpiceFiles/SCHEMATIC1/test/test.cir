** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "H:/UNI STUFF/YEAR 3/PROJECT/PSPICE/models/opa124_pspice_aio/opa124.lib" 
* From [PSPICE NETLIST] section of H:\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN I_I1 0 100p 0.1p 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
