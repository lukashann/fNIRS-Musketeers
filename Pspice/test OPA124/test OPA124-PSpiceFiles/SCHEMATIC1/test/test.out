
**** 11/18/16 15:54:43 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "H:/UNI STUFF/YEAR 3/PROJECT/PSPICE/models/opa124_pspice_aio/opa124.lib" 
* From [PSPICE NETLIST] section of H:\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN I_I1 0 100p 0.1p 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST OPA124
X_U3         0 N01887 N01899 N01891 N01895 OPA124
V_V3         N02033 0 5Vdc
I_I1         N018750 N01887 DC 100pAdc  
V_V1         N01899 0 15Vdc
V_V4         0 N02061 5Vdc
X_U2         N01895 N01993 N02033 N02061 N01993 LM358 
V_V2         0 N01891 15Vdc
C_C1         N01887 N01895  10p  TC=0,0 
R_R1         N01887 N01895  1G TC=0,0 
X_D1         0 N018750 D1N4500 

**** RESUMING test.cir ****
.END

**** 11/18/16 15:54:43 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U3.DY         X_U3.DX         X_D1.DFWD       X_D1.DXTRA      
          IS  380.000000E-18  800.000000E-18   71.949420E-15   68.819730E-12 
           N                                     .960183        2.15458      
         ISR                                   96.498840E-15                 
          NR                                    1.82557                      
         IKF                                  237.528000E-06     .053136     
          BV                                   86.542         100.000000E+03 
         IBV                                     .3447          1.000000E-03 
          RS                                     .236317         .028472     
          TT                                    6.656200E-09                 
         CJO                                    4.000000E-12                 
          VJ                                     .75                         
           M                                     .3333                       
          KF  117.500000E-18                                                 


               X_D1.DLEAK      X_U2.x_lm358.dx X_U2.x_lm358.dy 
          IS    5.810722E-15  800.000000E-18  800.000000E-18 
           N  180.584                                        
          BV  100.000000E+03                                 
         IBV    1.000000E-03                                 
          RS     .1             1               1.000000E-03 
         CJO                                   10.000000E-12 


**** 11/18/16 15:54:43 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U2.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 11/18/16 15:54:43 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Junction FET MODEL PARAMETERS


******************************************************************************




               X_U3.JX         
               PJF             
         VTO   -1            
        BETA     .01579      
          IS  400.000000E-15 


**** 11/18/16 15:54:43 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U2.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U2.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 11/18/16 15:54:43 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\UNI STUFF\YEAR 3\PROJECT\PSPICE\test OPA124\test OPA124-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .37
